fe/fe-core/src/main/java/com/starrocks/planner/DeltaLakeScanNode.java

```diff
@@ -281,6 +282,7 @@ protected void toThrift(TPlanNode msg) {
         HdfsScanNode.setScanOptimizeOptionToThrift(tHdfsScanNode, this);
         HdfsScanNode.setCloudConfigurationToThrift(tHdfsScanNode, cloudConfiguration);
         HdfsScanNode.setMinMaxConjunctsToThrift(tHdfsScanNode, this, this.getScanNodePredicates());
+        HdfsScanNode.setPartitionConjunctsToThrift(tHdfsScanNode, this, this.getScanNodePredicates());
```

Comment:
why only pass partition conjunct?  where is non-partition conjunct?

Reply:
non-partition conjunct is in plan node's conjucts